pkg/analysis/passes/screenshots/screenshots.go

```diff
@@ -46,6 +47,13 @@ func checkScreenshots(pass *analysis.Pass) (interface{}, error) {
 
 	var data metadata.Metadata
 	if err := json.Unmarshal(metadataBody, &data); err != nil {
+		// Check if the error is related to screenshots field format
```

Comment:
did we encounter an error here before? the reason this code is not present anywhere where we use the matadata is because this validator depends on `metadata.Analyzer`. if there's an error in the metadata json format it'll fail there and not here. 

in other words, we are guarantee to t least have a correct json here. not necessarily the correct schema, but correct json.